.github/workflows/build-sample-apps.yml

```diff
@@ -67,13 +67,37 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
+    - name: Set Default Firebase Distribution Groups
+      run: |
```

Comment:
I think we can extract the variable initialization from `run` to `env`, something like 

```
- name: Set Firebase Distribution Groups
  shell: bash
  env:
    # Distribution group constants
    ALL_BUILDS_GROUP: all-builds
    FEATURE_BUILDS_GROUP: feature-branch
    STABLE_BUILDS_GROUP: next
    # Input variables
    IS_PRIMARY_APP: ${{ matrix.apn-or-fcm == 'APN' }}
    CURRENT_BRANCH: ${{ github.ref }}
  run: |
    # Initialize array with default group
    distribution_groups=("$ALL_BUILDS_GROUP")
```

Reply:
Updated